src/modules/notifications/services/notifications.service.ts

```diff
@@ -167,6 +170,23 @@ export class NotificationsService {
 		return affectedRows;
 	}
 
+	public async delete(visitId: string, deleteNotificationDto: DeleteNotificationDto) {
+		const response = await this.dataService.execute<DeleteNotificationsMutation>(
+			DeleteNotificationsDocument,
+			{
+				where: {
+					visit_id: { _eq: visitId },
+					type: { _in: deleteNotificationDto.types },
```

Comment:
deleteNotificationDto.types is optional, what happens when you pass undefined?, shouldn't this default to either empty array or omit the where.type property?

Reply:
fixed